src/applications/vaos/services/appointment/index.js

```diff
@@ -176,9 +181,13 @@ export async function fetchRequestById({ id }) {
  * @param {Boolean} useV2 Toggle fetching VA or CC appointment via VAOS api services version 2
  * @returns {Appointment} A transformed appointment with the given id
  */
-export async function fetchBookedAppointment({ id, avs = true }) {
+export async function fetchBookedAppointment({
+  id,
+  avs = true,
+  fetchClaimStatus,
```

Comment:
Like AVS I'm okay with claims being defaulted to true for fetching single booked appointment.